app/views/sign_in/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render_component PageTitle::View.new(title: "sign_in.index") %>
```

Comment:
I'd probably call this `sessions#new`

Reply:
see https://github.com/DFE-Digital/register-trainee-teachers/pull/192#discussion_r527707887